include/global.h

```diff
@@ -719,8 +719,8 @@ struct LilycoveLadyQuiz
     /*0x000*/ u8 id;
     /*0x001*/ u8 phase;
     /*0x002*/ u16 unk_002[9];
-    /*0x014*/ u16 unk_014;
-    /*0x016*/ u16 unk_016;
+    /*0x014*/ u16 unk_014[1];
```

Comment:
Is this not unk_014[2], perhaps? This is just bizarre

Reply:
Don't ask me, it's Pikalax's.

Reply:
Try making the change and see if it works

Reply:
I just made it two seperate u16, like it was before.